sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -40,8 +40,8 @@
 #include <unordered_set>
 #include <vector>
 
-#include <level_zero/ze_api.h>
```

Comment:
Is this to fix the wrong include order at L0 build?
I am afraid this is just hiding the issue without fixing it.
The L0 is supposed to be included from <level_zero/

Reply:
No, this is needed to build based on level_zero project structure, rather than level_zero install structure.
Fix for includes order is CMAKE_INCLUDE_CURRENT_DIR OFF